connexion/apps/asynchronous.py

```diff
@@ -78,15 +79,17 @@ def make_operation(self, operation: AbstractOperation) -> AsyncOperation:
 
 
 class AsyncMiddlewareApp(RoutedMiddleware[AsyncApi]):
+
+    api_cls = AsyncApi
+
     def __init__(self) -> None:
         self.apis: t.Dict[str, AsyncApi] = {}
         self.operations: t.Dict[str, AsyncOperation] = {}
         self.router = Router()
         super().__init__(self.router)
 
     def add_api(self, *args, **kwargs):
-        api = AsyncApi(*args, **kwargs)
-        self.apis[api.base_path] = api
+        api = super().add_api(*args, **kwargs)
```

Comment:
Just a fix to get the `AsyncApp` working correctly. Not related to enforcing defaults, but necessary to make the example work.